app/main.py

```diff
@@ -1,13 +1,32 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
     pass
 
-
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
+    return {"errors":
+        [{format_linter_error(error)} for error in errors],
+    "path": file_path,
+    "status": "passed"
+    if errors == []
+    else "failed"
+    }
     pass
```

Comment:
The 'pass' statement is unnecessary here because the function already returns a value. You can remove it.